src/pages/register/Register.js

```diff
@@ -49,6 +53,32 @@ function Register() {
         onSubmit={(e) => handleRegistration(e)}
         className="registerContainer"
       >
+        <label>
+          First Name
+          <input 
+            type="text"
+            placeholder="Jane"
```

Comment:
placeholders can be removed from all inputs since we are adding labels. 
This [blog](https://www.smashingmagazine.com/2018/10/form-design-patterns-excerpt-a-registration-form/#:~:text=The%20label%20and%20placeholder%20text,to%20give%20users%20a%20hint.) elaborates on best practice in designing forms.

Reply:
Thanks, Alejandro! As per your request, I removed the placeholders from First and Last names labels.